packages/core/src/connection/__tests__/index.test.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * @jest-environment jsdom
+ */
```

Comment:
Declaring tests using this magic comment allows us to easily identify where our dom dependencies are